invenio_classifier/reader.py

```diff
@@ -254,10 +254,12 @@ def _discover_ontology(ontology_path):
     if not last_part.endswith('.rdf'):
         possible_patterns.append(last_part + '.rdf')
     places = [os.path.join(current_app.instance_path, "classifier"),
-              current_app.config['CLASSIFIER_WORKDIR'],
               os.path.abspath('.'),
               os.path.join(os.path.dirname(__file__), "classifier")]
 
+    if current_app.config['CLASSIFIER_WORKDIR'] is not None:
```

Comment:
more efficient approach to avoid double access to local proxy and config object would be:

```python
workdir = current_app.config.get('CLASSIFIER_WORKDIR')
if workdir is not None:  # or simply if workdir:
   places.append(workdir)
```